tracer/build/_build/docker/debian.dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG DOTNETSDK_VERSION
 # debian 10 image
-FROM mcr.microsoft.com/dotnet/sdk:$DOTNETSDK_VERSION-buster-slim as base
+FROM mcr.microsoft.com/dotnet/sdk:6.0.100-rc.2-bullseye-slim as base
```

Comment:
Same here, this shouldn't be hard coded

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/1885/commits/242c3c813e05991fdf1543aa6f7574ce577c6805 (I have to use a different variable while .NET 6 is in RC, because the image name is not the same as the full version name)